### PR TITLE
[Gateway] File sandboxing

### PR DESCRIPTION
**Pull request by maxvp**
PCX-12396

### CODE REVIEW

File: src/content/docs/cloudflare-one/policies/gateway/http-policies/antivirus-scanning.mdx [hunk 1]
```diff
@@ -2,111 +2,107 @@
...
+import { Render, Details } from "~/components";
 
-To prevent this, Cloudflare Gateway allows admins to enable **Anti-Virus (AV) scanning** of files that are uploaded or downloaded by users as the file passes through Gateway.
+Cloudflare Gateway protects users as they browse the Internet. When users download or upload a file to an origin on the Internet, that file could potentially contain malicious code that may cause their device to perform undesired behavior. To prevent this, Cloudflare Gateway allows admins to turn on anti-virus (AV) scanning of files that are uploaded or downloaded by users as the file passes through Gateway.
 
-AV scanning of files requires organizations to go to **Settings** > **Network** > **Firewall** and enable **AV inspection**.
+In addition to scanning files, Gateway can quarantine files as your users download them. Quarantining files helps protect organizations from zero-day vulnerabilities not yet available in anti-virus databases. For more information, refer to [File sandboxing](/cloudflare-one/policies/gateway/http-policies/file-sandboxing/).
```
Comment (Oxyjun) [inline src/content/docs/cloudflare-one/policies/gateway/http-policies/antivirus-scanning.mdx:12]:
```suggestion
In addition to scanning files, Gateway can quarantine files as your users download them. Quarantining files helps protect organizations from zero-day vulnerabilities not yet available in AV databases. For more information, refer to [File sandboxing](/cloudflare-one/policies/gateway/http-policies/file-sandboxing/).
```

Maybe it's better to use "AV" here, as we've already introduced the acronym?

File: src/content/docs/cloudflare-one/policies/gateway/http-policies/file-sandboxing.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+title: File sandboxing
+sidebar:
+  order: 6
+---
+
+import { Render, Details } from "~/components";
+
+In addition to [anti-virus (AV) scanning](/cloudflare-one/policies/gateway/http-policies/antivirus-scanning/), Gateway can quarantine previously unseen files downloaded by your users into a sandbox and scan them for malware.
+
+While the files are quarantined, Gateway will present a scanning page to your users. If a file passes scanning, Gateway will release the file from quarantine and download it to your user's device.
```
Comment (RebeccaTamachiro) [inline src/content/docs/cloudflare-one/policies/gateway/http-policies/file-sandboxing.mdx:12]:
```suggestion
While the files are quarantined, Gateway will present a scanning page to your users. If a file passes the scanning, Gateway will release the file from quarantine and download it to your user's device.
```

File: src/content/docs/cloudflare-one/policies/gateway/http-policies/antivirus-scanning.mdx [hunk 3]
```diff
@@ -2,111 +2,107 @@
...
 
 | Selector | Operator      | Value         | Action      |
 | -------- | ------------- | ------------- | ----------- |
 | Hostname | matches regex | `example.com` | Do Not Scan |
 
-Opting out of AV scanning applies to both uploads and downloads of files (i.e., it matches the global AV scanning setting). If an admin has chosen, for example, to only globally scan uploads, then opting out of AV scanning only applies to uploads.
+Opting out of AV scanning applies to both uploads and downloads of files, matching your account's global AV scanning setting. For example, if you have configured Gateway to globally scan uploads only, then opting out of AV scanning will only apply to uploads.
```
Comment (Oxyjun) [inline src/content/docs/cloudflare-one/policies/gateway/http-policies/antivirus-scanning.mdx]:
```suggestion
Opting out of AV scanning applies to uploads and/or downloads of files, matching your account's global AV scanning setting. For example, if you have configured Gateway to globally scan uploads only, then opting out of AV scanning will only apply to uploads.
```

At my first read-through I thought this sentence meant it applied to both uploads and downloads simultaneously - left a tiny suggestion using the `and/or` style seen in the text above the table.

**Review by Oxyjun — APPROVED** [review]

File: src/content/docs/cloudflare-one/policies/gateway/http-policies/antivirus-scanning.mdx [hunk 4]
```diff
@@ -2,111 +2,107 @@
...
-
-* If the Content-Disposition HTTP header is `Attachment`
-
-* If the byte signature of the body of the request matches a signature we identify as one of the following file type categories:
+## File scan criteria
 
-  * **Executable** (e.g., `.exe`, `.bat`, `.dll`, `.wasm`)
-  * **Documents** (e.g., `.doc`, `.docx`, `.pdf`, `.ppt`, `.xls`)
-  * **Compressed** (e.g., `.7z`, `.gz`, `.zip`, `.rar`)
+If AV scanning is turned on, Gateway will use the following to determine whether a file is present in a request or response, and whether to scan that file (first match will result in the file being scanned):
```
Comment (RebeccaTamachiro) [inline src/content/docs/cloudflare-one/policies/gateway/http-policies/antivirus-scanning.mdx]:
```suggestion
If AV scanning is turned on, Gateway will use the following criteria to determine whether a file is present in a request or response, and whether to scan that file. The first match will result in the file being scanned.
```

File: src/content/docs/cloudflare-one/policies/gateway/http-policies/index.mdx [hunk 5]
```diff
@@ -567,6 +595,12 @@ The country of the user making the request. <Render file="gateway/selectors/sour
 
 ## Logical operators
 
-<Render file="gateway/logical-operators" params={{ one: "**Identity** or **Device Posture**" }} />
+<Render
+	file="gateway/logical-operators"
+	params={{ one: "**Identity** or **Device Posture**" }}
```
Comment (RebeccaTamachiro) [inline src/content/docs/cloudflare-one/policies/gateway/http-policies/index.mdx]: Nit: I'm not entirely sure if it was intended or something with the partial, but the `**` are showing up in the rendered version

**Review by RebeccaTamachiro — APPROVED** [review]

**Pull request merged**